.github/workflows/publish.yml

```diff
@@ -11,9 +11,10 @@ jobs:
     if: github.repository == 'apache/incubator-pekko-sbt-paradox'
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
-          fetch-depth: 100
```

Comment:
Using a `fetch-depth` of 100 here would be wrong because the published version would end up being incorrect once the git history would get big enough (to the point where it wouldn't fetch the latest tag)